orc8r/lib/go/registry/global_registry.go

```diff
@@ -117,6 +117,12 @@ func GetServicePort(service string) (int, error) {
 	return globalRegistry.GetServicePort(service)
 }
 
+// GetProtectedServicePort returns the listening port for the protected RPC service.
+// The service needs to be added to the registry before this.
+func GetProtectedServicePort(service string) (int, error) {
```

Comment:
From offline: consider converting these to using the same funcs but with enums. This should help avoid a lot of duplicated code